ktor-http/ktor-http-cio/jvm/src/io/ktor/http/cio/RemoteHost.kt

```diff
@@ -0,0 +1,27 @@
+package io.ktor.http.cio
+
+import io.ktor.network.sockets.*
+import java.net.*
+import java.nio.channels.*
+
+fun remoteHost(socket: AConnectedSocket) = remoteHost(socket.remoteAddress)
```

Comment:
These functions look like they construct a remote host. More Kotlinish would be an extension property returning String

Reply:
Also, since it is a public API, it should be documented properly describing what does it return, when it is hostname or a stringified IP address.